lib/clients/BaseClient.py

```diff
@@ -531,6 +531,48 @@ def decrypt_and_extract_tarball_of_directory(self, encrypted_tarball_name, direc
             return result
         return None
 
+    def create_tarball_of_directory(self, directory_to_tar, tarball_name):
+        """Create a tarball of a directory.
+
+        :param directory_to_tar: the path to the directory to be archived
+        :param tarball_name: the path where to store the resulting archive
+
+        :Example:
+            ::
+                tar cvzpf ACS-Demo.tar.gz -C ACS-Demo .
+                iaas_client.create_tarball_of_directory('/tmp/backup/files.tar.gz', '/var/vcap/store/blueprint/files')
+        """
```

Comment:
Reverse the order "iaas_client.create_tarball_of_directory('/tmp/backup/files.tar.gz', '/var/vcap/store/blueprint/files')" of arguments.

Reply:
ok.just another way. I will do it

Reply:
Not just another way right? Your param list order is `directory_to_tar, tarball_name` and the example is in reverse order `'/tmp/backup/files.tar.gz', '/var/vcap/store/blueprint/files'`.